packages/@react-spectrum/s2/stories/Accordion.stories.tsx

```diff
@@ -34,15 +36,19 @@ export const Example: Story = {
         <Accordion {...args}>
           <Disclosure id="files">
             <DisclosureHeader>
-              Files
+              <DisclosureTrigger>
```

Comment:
This pattern is opposite to our other triggers
`<DialogTrigger>` is the outermost, whereas here, `DisclosureTrigger` is the innermost

Reply:
That's because the header can contain elements that are not part of the trigger (eg adjacent buttons).

Reply:
I understand that, but it also doesn't include the element that it opens like our other triggers
See my other concerns in my next comment down about `level`